editor/src/core/shared/element-path-tree.spec.browser2.tsx

```diff
@@ -136,6 +137,9 @@ describe('Building and ordering the element path tree for a real project', () =>
   })
 })
 describe('getChildrenOrdered', () => {
+  // This test only makes sense with the code in navigator FS off, otherwise the expression is the
```

Comment:
In that case I'd say it's worth adding a `Returns an expression child` test case that only runs with the FS on (unless we already have one)

Reply:
Done in 3a73678682918a4797f06f0b2a0058638209e804